aries-site/src/examples/cardPreviews/toast-notification.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StatusGoodSmall } from 'grommet-icons/icons/StatusGoodSmall';
+import { FormClose } from 'grommet-icons/icons/FormClose';
+
+import { Box, Button, Text } from 'grommet';
+
+export const ToastPreview = () => (
+    <Box align="center" justify="center" fill background="background-back">
```

Comment:
Will this get replaced with a `<Notification toast />` component once available from Grommet? If so, perhaps adding a TODO comment "replace this with ... when Notification component is available in Grommet" and/or add an issue to track this?

Reply:
Could you import Notification from `toast-notifications/ToastNotification.js` in order to re-use code? Looks very similar.

Basically thinking that if we are mimicking the implementation of the Notification component until it is available, can we mimic it in a single place, have all current imports point to it? Then when Notification is available, we simply replace the import from the local file to the import from Grommet?